src/static/js/modules/per-page-js-loader.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var jsManifest = require( '../config/js-manifest-config.js' );
+
+/**
+ * Load a page-specific JavaScript dynamically into the header
+ * if the page has a page-specific script available.
+ */
+function init() {
+  var jsLoader = require( '../modules/util/js-loader' );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/a28eaa72b807d831b51e1db642eba0678fc42aac